src/main/kotlin/dev/fakek/FakeContext.kt

```diff
@@ -31,6 +31,7 @@ class FakeContext(private val faker: Faker = Faker.instance()) {
     private val fakerColor by lazy { faker.color() }
     private val fakerInternet by lazy { faker.internet() }
     private val fakerName by lazy { faker.name() }
+    private val fakerBoolean by lazy { faker.bool() }
```

Comment:
I've been trying to keep properties organized by scope and alphabetically to better organize this class as we add more. There isn't a style guide for this project so not concerned about it right now, it's something I can always update in the future.